src/encoded/glossary.py

```diff
@@ -0,0 +1,23 @@
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPBadRequest
+from urllib.parse import (
+    parse_qs,
+    urlencode,
+)
+
+def includeme(config):
+    config.add_route('glossary', '/glossary{slash:/?}')
+    config.scan(__name__)
+
+@view_config(route_name='glossary', request_method='GET', permission='search')
+def glossary(context, request):
```

Comment:
Looks okay, just fake endpoint for the frontend to hook into?

Reply:
Yes, exactly